classes/game.rb

```diff
@@ -0,0 +1,23 @@
+class Game < Item
+  attr_reader :multiplayer, :last_played_at
+
+  def initialize(last_played_at, multiplayer)
```

Comment:
Thank you for your efforts. Could you please include the require_relative statement for the item file in this current file? This will allow you to use it as a parent of the game class.🤔🤔